src/App.js

```diff
@@ -10,13 +11,16 @@ const App = () => {
         <Header />
 
         <Container style={{ marginTop: '20px' }}>
-          <Row>
+          <Row style={{ alignItems: 'flex-start' }}>
```

Comment:
It'd be great to move styles into vars and even into a separate file. In that case, it'd be easier to reuse.